Conference-Room-Reservation-System/Conference-Room-Reservation-System/src/main/java/com/sda/conferenceroomreservationsystem/service/ConferenceRoomService.java

```diff
@@ -1,45 +1,49 @@
 package com.sda.conferenceroomreservationsystem.service;
 
-import com.sda.conferenceroomreservationsystem.exception.type.ConferenceRoomNotFoundException;
+import com.sda.conferenceroomreservationsystem.exception.ConferenceRoomNotFoundException;
+import com.sda.conferenceroomreservationsystem.mapper.ConferenceRoomMapper;
 import com.sda.conferenceroomreservationsystem.model.dto.ConferenceRoomDto;
 import com.sda.conferenceroomreservationsystem.model.entity.ConferenceRoom;
+import com.sda.conferenceroomreservationsystem.model.request.ConferenceRoomRequest;
 import com.sda.conferenceroomreservationsystem.repository.ConferenceRoomRepository;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 @Service
 @RequiredArgsConstructor
 public class ConferenceRoomService {
 
     private final ConferenceRoomRepository conferenceRoomRepository;
-    public List<ConferenceRoomDto> getAllConferenceRooms;
+    private final ConferenceRoomMapper conferenceRoomMapper;
 
 
-    public List<ConferenceRoomDto> getAllConferenceRooms() {
-        //TODO
-        return null;
+
+    public List<ConferenceRoomDto> getAll() {
+        return conferenceRoomRepository.findAll().stream().map(ConferenceRoomMapper::map).collect(Collectors.toList());
```

Comment:
I would simplify to "return conferenceRoomRepository.findAll().stream().map(ConferenceRoomMapper::map).toList();"